lang/java/reef-checkpoint/src/main/java/org/apache/reef/io/checkpoint/fs/FSCheckpointID.java

```diff
@@ -34,8 +34,7 @@
 
   private Path path;
 
-  public FSCheckpointID() {
```

Comment:
Spectacularly :)  The beginning of the stack trace says:
`Nov 07, 2016 8:11:50 PM org.apache.reef.examples.suspend.ObjectWritableCodec decode
SEVERE: Cannot decode class class org.apache.reef.io.checkpoint.fs.FSCheckpointID
java.lang.InstantiationException: org.apache.reef.io.checkpoint.fs.FSCheckpointID`